README.md

```diff
@@ -37,6 +37,9 @@ Available options:
   - `token_duration`: Validity period (in seconds) for retieved authorization tokens.
   - `aws_access_key_id`: Use a specific AWS access key to authenticate with AWS.
   - `aws_secret_access_key`: Use a specific AWS secret access key to authenticate with AWS.
+  - `assume_role`: Role ARN to assume with the current profile name to get the CodeArtifact credentials.
```

Comment:
I'd like this to be renamed to `assume_role_arn` to be more explicit about the content it should contain.